Ntuplizer/plugins/Ntuplizer.cc

```diff
@@ -245,7 +244,11 @@ Ntuplizer::~Ntuplizer()
 void Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup){
   
   nBranches_->reset();
-      
+
+  edm::Handle<reco::VertexCollection> vertices;
+  iEvent.getByToken(vtxToken_, vertices);
+  if( vertices->empty() ) return; // skip the event if no PV found
```

Comment:
Sorry, misread the code. Yes, no PV is crap and no point in analysing the event. Maybe there should be a log statement in this case with run and event number so that we at least have some kind of knowledge on this event, but it's not a must.